uint16 length
float64 goal_yaw
float64[] path_x
float64[] path_y
